qemu/tests/boot_with_max_mem.py

```diff
@@ -0,0 +1,32 @@
+from virttest import env_process, utils_misc
+
+
+def run(test, params, env):
+    """
+    Test steps:
+    1) Check whether the host memory is greater than boot guest memory:
+    If the host memory is greater than boot guest memory,
+    continue to test the case,Otherwise, skip the test case
+    2) Boot guest with max memory with two cpus
+    3) Check guest dmesg, no call trace message
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    host_mem = utils_misc.get_usable_memory_size() // 1024 // 1024
+    boot_mem = params.get("boot_mem")
+    if host_mem < int(boot_mem[0:-1]):
+        test.cancel("The host memory is less than %s of boot guest memory" % boot_mem)
```

Comment:
If add param `vm_mem_minimum = 8T` in cfg, L17~L20 can be removed,  framework can handle it automatically, cancel test when no enough mem with host.

Reply:
@yanan-fu , I already add param vm_mem_minimum = 8T in cfg, L17~L20 can be removed, please check it, thanks